pyaedt/hfss3dlayout.py

```diff
@@ -1,35 +1,4 @@
-"""
-Introduction
-------------------
-
-This class contains all HFSS 3D Layout functionalities. It inherits
-all objects that belong to HFSS 3D Layout, including EDB API queries.
-
-
-Examples
---------
-
-Create an ``Hfss3dLayout`` object and connect to an existing HFSS design or create a new HFSS design if one does not exist.
-
->>> aedtapp = Hfss3dLayout()
-
-Create an ``Hfss3dLayout`` object and link to a project named ``projectname``. If this project does not exist, create one with this name.
-
->>> aedtapp = Hfss3dLayout(projectname)
-
-Create an ``Hfss3dLayout`` object and link to a design named ``designname`` in a project named ``projectname``.
-
->>> aedtapp = Hfss3dLayout(projectname,designame)
-
-Create an ``Hfss3dLayout`` object and open the specified project.
-
->>> aedtapp = Hfss3dLayout("myfile.aedt")
-
-Create a ``Desktop on 2021R1`` object and then creates an ``Hfss3dLayout`` object and open the specified project.
-
->>> aedtapp = Hfss3dLayout(specified_version="2021.1", projectname="myfile.aedt")
-
-"""
+"""This module contains the ``Hfss3dLayout`` and ``SweepString`` classes."""
```

Comment:
When a module contains two classes, would you prefer the format you used here. Because some files have more than two classes, I've used this format:

This module contains these classes:``Hfss3dLayout`` and ``SweepString``.

However, I can easily change to your format when there are two occurrences::

Reply:
Either style is fine.  Importance is consistence (which I didn't pick up on...).

Reply:
```suggestion
"""This module contains these classes: ``Hfss3dLayout`` and ``SweepString``."""
```